### ISSUE TITLE
Saturation Slider not working

### ISSUE THREAD

**Issue by Knarzi** [derekhe/msfs2020-map-enhancement#789]
I'm trying to adjust the saturation of Bing latest maps, but the slider has no effect. Have set it to -50, but no changes. The others inclusive the RGB-Sliders are working.

**MSFS version:**
 - msstore 
 - MSFS2024

**Mod version:**
- 8.0.27

**Device ID**
- CJ8HH793RV28G896QG798ERXJY5V22BKFJ2W2GA152RCFBKGFG70

**Screenshots**

![image](https://github.com/user-attachments/assets/5013a030-dfc4-4bd6-b735-e071e558e43f)
![image](https://github.com/user-attachments/assets/44138ba4-d154-4178-8310-b4b728e18c58)

**Comment by derekhe** [comment]
You may have rolling cache data in the game. Please clear it before starting the game.

**Comment by Knarzi** [comment]
I delete the cache on every startup manualy. Is it possible that you are doing the color correction via the RGB slider and the general color adjustment via the saturation slider in one step? In 2024, the Bing maps have a strong blue cast, which was not the case in 2020, so I set the blue slider to -50, which cancels out the blue cast. But now I can no longer reduce the general saturation. I increased it above 0 as a test and it works. If you are really doing both in one step, this would have to be changed. First do the color correction with the RGB slider values and then take the final result and do another color correction with the saturation value on all color channels.
